src/sql/parts/notebook/notebook.component.ts

```diff
@@ -194,7 +194,7 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this._model.cells.forEach(cell => {
 			cell.trustedMode = isTrusted;
 		});
-		this.setDirty(true);
+		//TODO: Handle dirty for trust?
```

Comment:
Please add an issue to track this. Or can we just send a notebookModel.contentChanged event / a cell-level for trust changing so that this gets notified?